scripts/table_diff/changes/changes.go

```diff
@@ -66,40 +89,33 @@ func getColumnChanges(file *gitdiff.File, table string) (changes []change) {
 		}
 	}
 	for deletedName, deletedColumn := range deletedColumns {
-		if addedColumn, ok := addedColumns[deletedName]; ok {
-			if deletedColumn.dataType == addedColumn.dataType && deletedColumn.pk == addedColumn.pk {
-				changes = append(changes, change{
-					Text:     fmt.Sprintf("Table %s: column order changed for %s", backtickStrings(table, deletedName)...),
-					Breaking: false,
-				})
-				continue
-			}
-
-			if addedColumn.dataType != deletedColumn.dataType {
-				changes = append(changes, change{
-					Text:     fmt.Sprintf("Table %s: column type changed from %s to %s for %s", backtickStrings(table, deletedColumn.dataType, addedColumn.dataType, deletedName)...),
-					Breaking: false,
-				})
-			}
-
-			if addedColumn.pk && !deletedColumn.pk {
-				changes = append(changes, change{
-					Text:     fmt.Sprintf("Table %s: primary key constraint added to column %s", backtickStrings(table, deletedName)...),
-					Breaking: false,
-				})
-			}
-
-			if !addedColumn.pk && deletedColumn.pk {
-				changes = append(changes, change{
-					Text:     fmt.Sprintf("Table %s: primary key constraint removed from column %s", backtickStrings(table, deletedName)...),
-					Breaking: false,
-				})
-			}
-		} else {
+		addedColumn, ok := addedColumns[deletedName]
+		switch {
+		case !ok:
 			changes = append(changes, change{
 				Text:     fmt.Sprintf("Table %s: column removed %s from table", backtickStrings(table, deletedName)...),
 				Breaking: true,
 			})
+		case addedColumn.dataType != deletedColumn.dataType:
```

Comment:
I don't think this covers the same cases as before, for example what if `dataType` changes and a PK is added on the same column?
We will only report the first change instead of both, WDYT?

Also, this change seems unrelated to the purpose of this PR, so probably better to separate it as it makes harder to review the PR